build.gradle

```diff
@@ -129,7 +129,6 @@ dependencies {
     constraints {
         implementation('org.scala-lang:scala-library:2.13.13')
         implementation('org.json:json:20240205')
-        implementation('org.bitbucket.b_c:jose4j:0.9.5')
```

Comment:
This override was added for a jose4j vulnerability, as the old version of io.kubernetes brought in a vulnerable version. But with the upgrade, it is no longer needed.

```
> Task :dependencyInsight
org.bitbucket.b_c:jose4j:0.9.4
  Variant compile:
    | Attribute Name                 | Provided | Requested    |
    |--------------------------------|----------|--------------|
    | org.gradle.status              | release  |              |
    | org.gradle.category            | library  | library      |
    | org.gradle.libraryelements     | jar      | classes      |
    | org.gradle.usage               | java-api | java-api     |
    | org.gradle.dependency.bundling |          | external     |
    | org.gradle.jvm.environment     |          | standard-jvm |
    | org.gradle.jvm.version         |          | 17           |

org.bitbucket.b_c:jose4j:0.9.4
\--- io.kubernetes:client-java:20.0.1
     \--- compileClasspath